homebrew_challenge/src/components/HeatMap/Heatmap.css

```diff
@@ -77,3 +77,8 @@ tr.area td {
   border: 0;
   padding: 0;
 }
+
+.flex-container {
```

Comment:
I think this should identify the element you're styling to avoid unintentional effects:
 .heatmap tr.date-range td.flex-container

Reply:
I have amended and removed the date-range className that wasn't being used

Reply:
Fair enough, .heatmap td.flex-container then - wanting to avoid affecting *all* flex-container elements in the app. I'm not convinced the css is automatically partitioned by component after react is rendered.

Reply:
corrected